src/rpc/error.rs

```diff
@@ -31,6 +31,16 @@ impl ServerError {
     pub fn known_code(&self) -> ErrorCode {
         self.inner.code().into()
     }
+    pub fn stubbed_4528() -> Self {
```

Comment:
I don't think adding issues' numbers helps in code readability.
```suggestion
    pub fn stubbed() -> Self {
```

Reply:
Does it need to be issue specific? Can we have a common method for stubs (that may or may not accept optional issue for errors)? Otherwise, I wouldn't put in in a public API of the `ServerError`.

Reply:
> Can we have a common method for stubs

At the moment there's only one issue, if there are future methods which require stubbing and method tracking, I'm happy to have a common `stubbed` method.

> that may or may not accept optional issue for errors

Without the above, the callsite changes from `stubbed_4528` to `stubbed(4528)`, which I don't think addresses your initial comment

> I don't think adding issues' numbers helps in code readability.

> Otherwise, I wouldn't put in in a public API of the ServerError.

I don't know what you mean by this - I think it's helpful for us and users to have an issue number they can ping if they e.g want a specific method implemented.

---

How about I change this to `stubbed_for_openrpc`?

Reply:
> Without the above, the callsite changes from stubbed_4528 to stubbed(4528), which I don't think addresses your initial comment

This would actually address my concern. As a developer, I don't want to see a `ServerError::stubbed_XYZ`. I want to see only methods that are usable for me.


> I don't know what you mean by this - I think it's helpful for us and users to have an issue number they can ping if they e.g want a specific method implemented.

I agree it's helpful - in the logs, return messages, but on in the public interface of the `ServerError`.